### ISSUE TITLE
Crash under special circumstances

### ISSUE THREAD

**Issue by ModProg** [ModProg/BlockMeter#21]
Crash Report:
https://pastebin.com/q6w6rRip

**Comment by SrFodonis** [comment]
Installed mods in when the crash occurred: https://pastebin.com/biSKwHuJ

**Comment by ModProg** [comment]
I will close this for now I assume ot is related to some incompatible configuration format

**Comment by Oyssis** [comment]
yeah deleting blockmeter.toml fixes it but every time you start your game it does that

**Comment by ModProg** [comment]
That is strange, it should only happen when the file is corrupt or maybe of a different version.

Do you use two different versions of Blockmeter?

**Comment by Oyssis** [comment]
it just crashed again and no I do not use two different versions of Blockmeter

**Comment by Oyssis** [comment]
[crash-2022-10-14_08.32.00-client.txt](https://github.com/ModProg/BlockMeter/files/9782771/crash-2022-10-14_08.32.00-client.txt)
the logs do seem a bit different

**Comment by Oyssis** [comment]
weird.. its empty..
